src/app/search/search.component.html

```diff
@@ -20,3 +20,13 @@ <h1>Select an Artifact</h1>
     </div>
   </div>
 </div>
+
+<div class="showNotification" *ngIf="errorMsg">
+  <pfng-toast-notification
+    [header]="'Error!'"
```

Comment:
How about `Error:`? Seems less scary :)

We could also just drop the header entirely. It's obvious it's an error since it's red.